tests/test_autonomy/test_replay.py

```diff
@@ -45,28 +45,15 @@
     "mem_reservation": "256M",
     "cpus": 1,
     "container_name": "abci0",
-    "image": "valory/open-autonomy-open-aea:oracle-0.1.0",
+    "image": "valory/open-autonomy-open-aea:hello-world-0.1.0",
     "environment": [
         "LOG_FILE=/logs/aea_0.txt",
         "ID=0",
-        "VALORY_APPLICATION=valory/oracle:0.1.0:QmUTuBXUbgBjBbiLR3kmBxZXQowseu2mc6xR5MVuX8UyXJ",
+        "VALORY_APPLICATION=valory/hello_world:0.1.0:bafybeideb6b5k4i6z7bm3p53eydxgknmwdefo2oshcnlxthjc6oxeox7ua",
         "ABCI_HOST=abci0",
         "MAX_PARTICIPANTS=4",
         "TENDERMINT_URL=http://node0:26657",
         "TENDERMINT_COM_URL=http://node0:8080",
-        "SKILL_ORACLE_ABCI_MODELS_PRICE_API_ARGS_URL=https://api.coingecko.com/api/v3/simple/price",
```

Comment:
Is it safe to remove this for the test to retain meaning?

Reply:
I believe yes, but we would need @angrybayblade's input too.